modules/ROOT/pages/basketball-robot-guide/mechanics/milling.adoc

```diff
@@ -0,0 +1,430 @@
+
+The mill has no power when the Emergency Stop button is engaged. +
+The button can be disengaged by twisting it clockwise until it pops out.
+
+
+=== 4. Launch UCCNC
+
+
+=== 5. Disable Reset in UCCNC
+
+
+=== 6. Perform Double-Homing action
+
+This will move the mill all the way to the end of each axis, calibrating the position of the mill and
+giving you room to place the workpiece on the table.
+
+
+=== 7. Make sure that the mill can move around in XYZ-axis
+
+The mill should be freely movable around the workarea.
+
+
+=== 8. Clean up your material
+
+Remove plastic coverings off your material and brush over your material to remove debris.
+
+
+=== 9. Place pieces of double-sided tape on the bottom of the material
+
+Do not remove the coverings off the double-sided tape pieces.
+The tape should be attached to the extremities (corners and edges)
+and in the region where your parts will be milled out.
+Empty regions should also have a few pieces of tape.
+When placing the tape, make sure that the corners of the pieces don't go under themselves.
+
+
+=== 10. Remove the coverings off the double-sided tape
+
+Lay your material on a flat surface. Remove the coverings off the tape pieces.
+
+It's better to work your way from the back to the front when removing the covering off the tape pieces.
+
+
+=== 11. Place your workpiece on the table
+
+Make sure to properly align the piece.
+The workpiece must not hang over the table and should be on the spoilboard.
+
+A line is drawn on the spoilboard which can be used to align your material.
+
+
+=== 12. Insert your first tool into the spindle
+
+image::mill_collet.png[]
+
+* Check that the collet is properly attached to the clamping nut.
+** If the collet is loose, then it should be attached, without a tool inserted, into the clamping nut.
+An audible click should be heard.
+
+* The tool should be in the collet as deep as possible,
+but such that the collet is not covering the cutting edges of the tool.
+** Leaving 1~2 mm between the cutting edge and the collet is enough.
+
+* When attaching the tool, the locking button of the spindle has to be pressed in.
+* Screw the clamping nut onto the spindle as much as you can with fingers, then use a 17 mm wrench. +
+* Do not attach the clamping nut very tightly - when you later have to remove the collet, using too much force
+can cause you to jerk the machine and the axis can move out of place.
+
+=== 13. Zero the X and Y axis
+
+image::mill_xy_zero.png[]
+
+The zero point should be set based on the location of your Work Coordinate System in your CAM.
+Set the zero point to be 5 mm from the edge of the workpiece when
+a 3 mm diameter tool would be attached to the spindle
+
+
+=== 14. Check that your G-code fits on the workpiece
+
+Load in the final outer contour milling G-code and go over the workpiece with the mill,
+referencing the screen in UCCNC.
+You can see when the tool goes over the contour, if it still stays on the workpiece.

```

Comment:
I would connect the two sentences with a statement and then a reason. Currently it reads as 2 separate statements.